File: ios/tests_class_sel_imp/web/Bull.js
var BullParent = Cattle;
var BullParentProto = BullParent.prototype;
function Bull()
{
	BullParent.call(this);
	this.skinColor = null;
}

for(var selector in BullParentProto){
	if(BullParentProto.hasOwnProperty(selector)){
		Bull.prototype[selector] = BullParentProto[selector];
	}
}

Bull.prototype.saySomething = function saySomething()
{
	console.log("Hello, I am a " + this.skinColor + " bull, I have " + this.legsCount + " legs.");
}

Bull.prototype.getSkinColor = function getSkinColor()
{
	return this.skinColor;
}

Bull.prototype.setSkinColor = function setSkinColor(color)
{
	this.skinColor = color;
}

File: ios/06-NSObject/06-NSObject/Bull.m
//
//  Bull.m
//  tests_class_sel_imp
//
//  Created by 裴超 on 13-5-8.
//  Copyright (c) 2013年 裴超. All rights reserved.
//

#import "Bull.h"

@implementation Bull

-(void) saySomething
{
	NSLog(@"Hello, I am a %@ bull, I have %d legs.", [self getSkinColor], legsCount);
}

-(NSString *) getSkinColor
{
	return skinColor;
}

-(void) setSkinColor:(NSString *)color
{
	skinColor = color;
}

@end
